File: app/src/main/java/com/ishow/noahark/modules/base/AppBaseFragment.java
package com.ishow.noahark.modules.base;

import com.baidu.mobstat.StatService;
import com.ishow.common.app.fragment.BaseFragment;

/**
 * 基础Fragment
 * 备用：
 */
public class AppBaseFragment extends BaseFragment {
    private static final String TAG = "AppBaseFragment";

    @Override
    public void onPause() {
        super.onPause();
        // 页面埋点
        StatService.onPageEnd(getActivity(), null);
    }

    @Override
    public void onResume() {
        super.onResume();
        // 页面埋点
        StatService.onPageStart(getActivity(), null);
    }

}

File: app/src/main/java/com/ishow/noah/modules/base/AppBaseFragment.kt
package com.ishow.noah.modules.base

import com.baidu.mobstat.StatService
import com.ishow.common.app.fragment.BaseFragment

/**
 * Created by yuhaiyang on 2018/8/8.
 * 基础Fragment
 * 备用：
 */
abstract class AppBaseFragment : BaseFragment() {

    override fun onPause() {
        super.onPause()
        // 页面埋点
        StatService.onPageEnd(activity, null)
    }

    override fun onResume() {
        super.onResume()
        // 页面埋点
        StatService.onPageStart(activity, null)
    }


}
